models/point_of_interest.py

```diff
@@ -38,6 +41,31 @@ def ensure_categories_as_list(cls, categories: Any) -> List[str]:
     def ensure_organizations_as_list(cls, organizations: Any) -> List[str]:
         return _convert_to_list(organizations)
 
+    @validator('url', pre=True)
+    def sanitize_url(cls, url: str) -> str:
+        org_url = url
+        if len(url.split('.')) < 3 and url.startswith('www') or '@' in url:
+            return ERROR_PREFIX + org_url
+
+        markdown_url = re.search('\[.*\]\((.*)\)', url)
```

Comment:
You're taking the `[]` part of markdown? You should take the `()` part coz it could be like `[Google](https://google.com)`

Reply:
No.

I extract the URL in a Markdown format, I included `[]` with `()` to make sure it's a Markdown.
And I'm sorry since it's not readable, more detail please check https://regexr.com/6o1jf